BosTau.py

```diff
@@ -0,0 +1,211 @@
+"""
+Catalog definitions for Bos Taurus
+"""
+import math
+import logging
+
+import msprime
+
+import stdpopsim
+
+logger = logging.getLogger(__name__)
+
+###########################################################
+#
+# Genome definition
+#
+###########################################################
+
+
+# Create a string of chromosome lengths for easy parsing
+_chromosome_data = """\
+chr1    158534110
+chr2    136231102
+chr3    121005158
+chr4    120000601
+chr5    120089316
+chr6    117806340
+chr7    110682743
+chr8    113319770
+chr9    105454467
+chr10   103308737
+chr11   106982474
+chr12   87216183
+chr13   83472345
+chr14   82403003
+chr15   85007780
+chr16   81013979
+chr17   73167244
+chr18   65820629
+chr19   63449741
+chr20   71974595
+chr21   69862954
+chr22   60773035
+chr23   52498615
+chr24   62317253
+chr25   42350435
+chr26   51992305
+chr27   45612108
+chr28   45940150
+chr29   51098607
+chrX    139009144
+chrMT   16338
```

Comment:
It's probably best to remove chrMT for now, as we don't currently have special-case suport for mitochondria. If someone simulates with chrMT, it will erroneously use the recombination rate given below.